src/js/game/LevelMVC/LevelView.js

```diff
@@ -536,48 +537,47 @@ module.exports = class LevelView {
     }
   }
 
-  playMinecartAnimation(position, facing, isOnBlock, completionHandler, minecartTrack, unpoweredRails) {
-    var animation;
-    this.track = minecartTrack;
-    this.i = 0;
-
+  playMinecartAnimation(isOnBlock, completionHandler, unpoweredRails) {
     //start at 3,2
     this.setPlayerPosition(3, 2, isOnBlock);
-    position = [3, 2];
+    const position = [3, 2];
+    this.player.facing = 2;
```

Comment:
I'm surprised to still see this here; shouldn't it be generalized away as a part of this PR?

Reply:
Unfortunately this animation is specific to the Adventurer 13 minecart level.  The player enters the house facing north, but in the cut scene animation we move them to location [3, 2] and make them face south.  Then we start calling `playTrack` to make them drive off screen.